src/layers/invertible_layer_conv1x1.jl

```diff
@@ -159,8 +159,8 @@ function conv1x1_grad_v(X::AbstractArray{T, N}, ΔY::AbstractArray{T, N},
     n_in, batchsize = size(X)[N-1:N]
     prod_res = cuzeros(X, size(dV1, 1))
     for i=1:batchsize
-        Xi = -2f0*reshape(selectdim(X, N, i), :, n_in)
-        ΔYi = reshape(selectdim(ΔY, N, i), :, n_in)
+        Xi = -2*reshape(selectdim(X, N, i), :, n_in)
+        ΔYi = 1*reshape(selectdim(ΔY, N, i), :, n_in) # 1* force conversion from ReshapedArray to array
```

Comment:
Why do you need that conversion? that's an extra allocation for a full tensor no?

Reply:
True, but strangely enough, otherwise this would give me an error in the function `mat_tens_i` function at the line `copyto!(out, map(i -> dot(Mat * Tens[i, :, :], Mat2) , 1:size(Tens, 1)))`. Here `Mat2` would be the variable `ΔYi` above, and `dot` doesn't like the `ReshapedArray`.

Reply:
By the way, this seems a problem only for julia 1.9.*, julia 1.8.5 doesn't complain :S

Reply:
Now that you mention it it does ring a bell, I think I had some combination of dependencies raising that issue at some point

Reply:
`collect` is slightly cheaer though it doesn't call an extra `mul` but it's minimal difference